Common/TDSQASystemAPI/Utilities/ValidationError.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace TDSQASystemAPI.Utilities
+{
+    /// <summary>
+    /// A generic error object, similar in structure to the <code>ValidationError</code> in TDS.
+    /// </summary>
+    public class ValidationError
+    {

```

Comment:
This class is similar to what we return in TDS.  The difference is the `severity` field, which we have in support tool's version of the `ValidationError`, but not in the `tds-common-legacy` version.  Since we discussed returning a `ValidationError`-like response for loading a test package into TIS, this is what I went with.